### ISSUE TITLE
add is_in() assertion

### ISSUE THREAD

**Issue by saturnboy** [ActivisionGameScience/assertpy#28]
For comparing the given val against a set of arguments:

```py
assert_that(1).is_in(1,2,3)
assert_that('foo').is_in('foo', 'bar', 'baz')
assert_that([1,2,3]).is_in([1,2,3], [2,3,4], [3,4,5])
```

Passes if and only if the given val is exactly equal to one of the given args.

**Comment by saturnboy** [comment]
The *not* variant `is_not_in()` should work like this:

```py
assert_that(4).is_not_in(1,2,3)
```

**Comment by saturnboy** [comment]
fixed by commit 470c390
